e2e/Dockerfile

```diff
@@ -3,7 +3,7 @@
 #FROM cypress/base:10
 
 # See the available Docker images here: https://github.com/cypress-io/cypress-docker-images/tree/master/browsers
-FROM cypress/browsers:node13.6.0-chrome80-ff72
+FROM cypress/browsers:node14.7.0-chrome84
```

Comment:
Should we also consider updating `node` to 14 elsewhere ? I think now Jenkins is running 13.6

Reply:
Yeah, probably in the near future. In this case I had to use the recommended image from cypress as it contains also the dependencies needed for Cypress 5.0.0 to run